task/github-create-deployment-status/0.1/README.md

```diff
@@ -0,0 +1,67 @@
+## GitHub Create Deployment Status
+
+The `github-create-deployment-status` Task lets you create a status for GitHub deployment.
+
+See GitHub's deployment API on [Create a deployment status](https://docs.github.com/rest/reference/repos#create-a-deployment-status) for more information.
+
+### Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/github-create-deployment-status/0.1/github-create-deployment-status.yaml
+```
+
+### Secrets
+
+This Task requires access to a GitHub token set via a Kubernetes Secret. By default, the name of this Secret should be `github` and the secret key should be `token`, but you can configure this via the `GITHUB_TOKEN_SECRET_NAME` and `GITHUB_TOKEN_SECRET_KEY` [parameters](#parameters) described below.
+
+To create such a Secret via `kubectl`:
+
+```
+kubectl create secret generic github --from-literal token="MY_TOKEN"
+```
+
+See GitHub's documentation on [Scopes for OAuth Apps](https://docs.github.com/developers/apps/scopes-for-oauth-apps) to figure out what scopes you need to give to this token to create deployments.
```

Comment:
apart from this can we add the scopes which are mandatory for this task. I guess below scopes are required. Please do correct if I am wrong :sweat_smile: 
```
Users with push access can create deployment statuses for a given deployment.

GitHub Apps require read & write access to "Deployments" and read-only access to "Repo contents" (for private repos). OAuth Apps require the repo_deployment scope.
```

Reply:
I agree with adding scopes to documentation, but I'm not sure about mentioning GitHub apps in this document.

Most GItHub apps will use installation access token, which [expires after one hour](https://docs.github.com/developers/apps/authenticating-with-github-apps#authenticating-as-an-installation), I think it's not suitable for this task since it requires to manage access token as `Secret` resource 🤔

(JFYI, our team patches GitHub related tasks in catalog to pass installation access token as parameter)